### PR TITLE
Revert and format latest commit.

### PR DESCRIPTION
**Pull request by Hexiaoqiao**
Update comments,
a. revert the latest commit which is mis-format.
b. re-commit and update two committer's employer info which latest commit want to do.

### CODE REVIEW

File: content/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-	<meta name="generator" content="Hugo 0.121.1">
+	<meta name="generator" content="Hugo 0.108.0">
```
Comment (ayushtkn) [inline content/index.html]: Why are you changing the hugo version
https://github.com/apache/hadoop-site/blob/asf-site/content/index.html#L6

Comment (aajisaka) [reply content/index.html]: It would be better to update the Hugo version in a separate PR, but personally I'm okay with that.

File: src/who.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ order):
   jitendra         |[Jitendra Nath Pandey](http://people.apache.org/~jitendra)                   |Hortonworks    |            |-8
   jlowe            |Jason Lowe                                                                   |NVIDIA         |            |-6
   jzhuge           |John Zhuge                                                                   |Netflix        |            |-8
-  junping\_du      |Junping Du                                                                   |Datastrato     |            |-8
+  junping\_du      |Junping Du                                                                   |Datastrato     |            |+8
```
Comment (aajisaka) [inline src/who.md:82]: Would you update the who.md in a separate PR?

File: content/index.html (hunk 1 above)
Comment (Hexiaoqiao) [reply content/index.html]: before 6c88b68c2c7225e606bfc10cca6dee32b6a5bf6b, the Hugo version is 0.111.3. It seems my local hugo version is not match to repo, I would like to update and re-submit.

File: src/who.md (hunk 2 above)
Comment (Hexiaoqiao) [reply src/who.md:82]: If need I can try to separate them. At this PR, I try to submit different commits for the three changes at one PR as description says.

Comment (Hexiaoqiao) [reply src/who.md:82]: ref: https://github.com/apache/hadoop-site/pull/49/commits

**Comment by Hexiaoqiao** [comment]
Hi @aajisaka @ayushtkn I try to format the latest commit first, I will try to remove release 3.2.4 at next PR.

**Review by ayushtkn — APPROVED** [review]
LGTM

**Review by aajisaka — APPROVED** [review]

**Comment by Hexiaoqiao** [comment]
Thanks @ayushtkn and @aajisaka . Merged.

**Pull request merged**